src/client.rs

```diff
@@ -1434,7 +1428,7 @@ fn digest_header_value(headers: HeaderMap, body: Option<&str>) -> Result<String>
 }
 
 /// Computes the SHA256 digest of a byte vector
-fn sha256_digest(bytes: &[u8]) -> String {
+pub(crate) fn sha256_digest(bytes: &[u8]) -> String {
```

Comment:
This probably belongs at the top level `lib.rs` as it is used in multiple places and doesn't have much to do with the client itself

Reply:
good point, done